### ISSUE TITLE
World Quest 42926 Vault of the Wardens bug

### ISSUE THREAD

**Issue by VILRIETTA** [dufernst/LegionCore-7.3.5#45]
This quest should be given when entering the dungeon, but due to an incorrect AreaGroupID in quest_template, it is not given, my fix is to fix this).
SQL:
UPDATE `quest_template` SET `AreaGroupID`='4803' WHERE (`ID`='42926')

**Comment by dufernst** [comment]
@VILRIETTA I would be happy to include this fix, it would save my some time if you could link me to some "proof" that this is indeed the correct way to fix this issue.

**Comment by VILRIETTA** [comment]
If I understand correctly how the world quest system works in this emulator, it should work correctly, because in this emulator the world quest system works on the principle of finding the AreaGroupID and comparing it with the AreaGroupID in which the player is located.In the database, the quest_template table has a column called AreaGroupID. IDs are taken from the RelationshipData column located in AreaGroupMember.db2

**Comment by VILRIETTA** [comment]
![изображение](https://user-images.githubusercontent.com/83026365/153067087-830754ee-c6e1-4715-9d22-b4697ee308c5.png)
dungeon map id 7787 this id corresponds to AreaGroupID 4803
Zone ID = map id
![изображение](https://user-images.githubusercontent.com/83026365/153067354-abff919a-5c2b-4eaf-9734-30d4f4477bc8.png)

**Comment by eXPeRi91** [comment]
I agree with @VILRIETTA but I don’t understand why people from uwow will use different AreaGroupID.

**Comment by ImmortalusDev** [comment]
This SQL fix is correct

**Comment by dufernst** [comment]
Thanks for the confirmations.
